src/main/groovy/com/google/protobuf/gradle/ProtobufPlugin.groovy

```diff
@@ -489,21 +490,60 @@ class ProtobufPlugin implements Plugin<Project> {
      * Adds proto sources and generated sources to supported IDE plugins.
      */
     private void addSourcesToIde() {
-      // Make the proto source dirs known to IDEs
-      getSourceSets().each { sourceSet ->
-        ProtobufSourceDirectorySet protoSrcDirSet = sourceSet.proto
-        protoSrcDirSet.srcDirs.each { File protoDir ->
-          Utils.addToIdeSources(project, Utils.isTest(sourceSet.name), protoDir)
+      // The generated javalite sources have lint issues:
+      //   https://github.com/google/protobuf/pull/2823
```

Comment:
Aren't these issues already fixed?
Besides, lint warnings don't seem to be a strong enough case to justify a new flag.

Reply:
The latest javalite artifact is still from 2016, so there's no release with the fix yet:
http://search.maven.org/#search%7Cga%7C1%7Ca%3A%22protoc-gen-javalite%22